src/goMain.ts

```diff
@@ -57,6 +57,15 @@ export function activate(ctx: vscode.ExtensionContext): void {
 		vscode.window.showInformationMessage('Current GOPATH:' + gopath);
 	}));
 
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.kill.running', () => {
+		killRunning();
+	}));
+
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.test.cursor', () => {
```

Comment:
The `go.test.cursor` command is already registered [here](https://github.com/Centny/vscode-go/blob/fd9d200269c632b68987463c2ead869eca0e69f6/src/goMain.ts#L64).